tasks/35_array-leaders/array-leaders.js

```diff
@@ -0,0 +1,13 @@
+function arrayLeaders(input) {
+  const arr = input.slice();
+  const leaders = [];
+  arr.reduce(function(sum, item) {
+    if (item > sum) {
+      leaders.push(item);
```

Comment:
Looks like this is not the case for `reduce()` method. Using an outer variable is the marker.

Reply:
Usually `reduce()` is applied when you need to iterate over the array and calc some outcome. As far as I can see there is no resulting assigned value